github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRTriggerMode.java

```diff
@@ -6,10 +6,10 @@
  * @author Kanstantsin Shautsou
  */
 public enum GitHubPRTriggerMode {
-    CRON ("Cron with persistence"),
-    HEAVY_HOOKS ("Experimental: Hooks with persistence"),
-    HEAVY_HOOKS_CRON ("NOT SUPPORTED: Hooks with persistence and periodic cron check"),
-    LIGHT_HOOKS("NOT SUPPORTED: Hooks without persistence");
+    CRON ("Cron"),
+    HEAVY_HOOKS ("Hooks"),
+    HEAVY_HOOKS_CRON ("NOT SUPPORTED: Hooks plus Cron"),
+    LIGHT_HOOKS("NOT SUPPORTED: Transient Hooks");
```

Comment:
My idea was that 'heavy' hooks do additional calls to github and persist state into xml. While light (like CB PR plugin) based only on incoming hook with limited info and doesn't do additional calls to GH.